src/Action/Admin/OrderRefundsListAction.php

```diff
@@ -20,40 +20,27 @@
 use Sylius\RefundPlugin\Provider\RefundPaymentMethodsProviderInterface;
 use Symfony\Component\HttpFoundation\RedirectResponse;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
 use Symfony\Component\HttpFoundation\Response;
-use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
 use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 use Twig\Environment;
 use Webmozart\Assert\Assert;
 
 final class OrderRefundsListAction
 {
-    private OrderRepositoryInterface $orderRepository;
-
-    private OrderRefundingAvailabilityCheckerInterface $orderRefundsListAvailabilityChecker;
-
-    private RefundPaymentMethodsProviderInterface $refundPaymentMethodsProvider;
-
-    private Environment $twig;
-
-    private Session $session;
-
-    private UrlGeneratorInterface $router;
-
     public function __construct(
-        OrderRepositoryInterface $orderRepository,
-        OrderRefundingAvailabilityCheckerInterface $orderRefundsListAvailabilityChecker,
-        RefundPaymentMethodsProviderInterface $refundPaymentMethodsProvider,
-        Environment $twig,
-        Session $session,
-        UrlGeneratorInterface $router
+        private OrderRepositoryInterface $orderRepository,
+        private OrderRefundingAvailabilityCheckerInterface $orderRefundsListAvailabilityChecker,
+        private RefundPaymentMethodsProviderInterface $refundPaymentMethodsProvider,
+        private Environment $twig,
+        private SessionInterface | RequestStack $requestStackOrSession,
```

Comment:
```suggestion
        private SessionInterface|RequestStack $requestStackOrSession,
```

Reply:
👍️ but our ECS config forces spaces here because for some reason it thinks it's a bitwise operator. I'm going to fix it in [SyliusLabs/CodingStandard](https://github.com/SyliusLabs/CodingStandard)